### PR TITLE
Orionjs

### PR DESCRIPTION
**Pull request by zhenzhongxu**
integrate with orionjs cms
implemented simple PoC volunteering collection
added dev init package
routing works with angular

todo: add angular router so orionjs pages does not use default template

### CODE REVIEW

File: client/app.html [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
-</body>
\ No newline at end of file
+<body>
+    <div>Hello world @body</div>
+</body>
+
+
+<template name="index" data-ng-app="UniversalGiving">
+    <navbar></navbar>
+    <div ui-view></div>
```
Comment (garukun) [inline client/app.html]: It's always a good practice to prepend `data-` for non-standard HTML attributes. So, this one should be: 

``` html
<div data-ui-view></div>
```

File: client/router.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Router.route('/', function(){
```
Comment (garukun) [inline client/router.js]: `app*` should be the only file in the root directory, so I would probably put this one in `lib` if it still works.

File: .meteor/versions [hunk 3]
```diff
@@ -33,35 +46,76 @@ htmljs@1.0.5
 http@1.1.1
 id-map@1.0.4
 insecure@1.0.4
+iron:controller@1.0.8
+iron:core@1.0.8
+iron:dynamic-template@1.0.8
+iron:layout@1.0.8
+iron:location@1.0.9
+iron:middleware-stack@1.0.9
+iron:router@1.0.9
```
Comment (garukun) [inline .meteor/versions:55]: Okay, do we really need iron router right now? Is this a dependency for the `orion`?

Comment (zhenzhongxu) [inline .meteor/versions:55]: yes, it's a hard dependency for orion.

File: client/router.js (hunk 2 above)
Comment (zhenzhongxu) [inline client/router.js]: moved to /lib

File: client/app.html (hunk 1 above)
Comment (zhenzhongxu) [inline client/app.html]: fixed

**Pull request merged**